src/screens/BB-BUS/DetailScreen.js

```diff
@@ -40,24 +41,22 @@ const TEXT_BLOCKS_SORTER = {
   'Zuständige Stelle': 15
 };
 
-const FormButton = ({ link, rootRouteName }) => {
+const FormButton = ({ link, name }) => {
   const { url } = link;
 
-  const openWebScreen = useOpenWebScreen(name, url, rootRouteName);
-
-  return <Button title={url} onPress={openWebScreen} invert />;
+  return <Button title={name} onPress={() => openURL(url)} invert />;
```

Comment:
i think we should use `openLink` from our `linkHelper`, that wrapps `openURL` from `expo-linking` with some additional stuff

Reply:
✅